specutils/wcs/adapters/fitswcs_adapter.py

```diff
@@ -0,0 +1,31 @@
+from ..wcs_adapter import WCSAdapter, WCSAxes
+
+from astropy.wcs import WCS
+from astropy.wcs import WCSSUB_SPECTRAL, WCSSUB_LONGITUDE, WCSSUB_LATITUDE, \
+    WCSSUB_CUBEFACE, WCSSUB_STOKES, WCSSUB_CELESTIAL
+
+
+__all__ = ['FITSWCSAdapter']
+
+
+class FITSWCSAdapter(WCSAdapter):
+    wrapped_class = WCS
+    axes = None
+
+    def __init__(self, wcs):
+        super(FITSWCSAdapter, self).__init__(wcs)
+
+        self.axes = WCSAxes(
+            longitude=self.wcs.sub([WCSSUB_LONGITUDE]),
+            latitude=self.wcs.sub([WCSSUB_LATITUDE]),
+            cubeface=self.wcs.sub([WCSSUB_CUBEFACE]),
+            spectral=self.wcs.sub([WCSSUB_SPECTRAL]),
+            stokes=self.wcs.sub([WCSSUB_STOKES]),
+            celestial=self.wcs.sub([WCSSUB_CELESTIAL])
```

Comment:
In this case, each `axes` object is a complete WCS object of its own.  Is that what we want?  Should these perhaps also be "adapters", e.g. `longitude=FITSWCSAdapter(self.wcs.sub([WCSSUB_LONGITUDE]))`?